config/runtime.exs

```diff
@@ -181,7 +180,7 @@ if config_env() == :prod do
     wireguard_public_key: wireguard_public_key,
     wireguard_interface_name: wireguard_interface_name,
     wireguard_port: wireguard_port,
-    cli: FzVpn.CLI.Live
+    wg_adapter: FzVpn.Interface.WGAdapter.Live
```

Comment:
I think you want to move this to `prod.exs` and leave this file for runtime configuration, e.g. through env vars. That'll hopefully keep things a bit tidier.

Reply:
(don't think we need to set Adapter at runtime -- only compile time, so can stay in the other config files)